@here/harp-mapview/lib/MapView.ts

```diff
@@ -2224,7 +2349,7 @@ export class MapView extends THREE.EventDispatcher {
      * in the [[MapViewOptions]]
      */
     renderSync() {
-        this.renderFunc(PerformanceTimer.now());
+        this.render(PerformanceTimer.now());
```

Comment:
@zbigg @ninok I never understood why we had `renderFunc` there and what it was. Does it serve any purpose?

Reply:
looks fine to me to remove it.